### PR TITLE
[localize] graphs.h

### PR DESCRIPTION
**Pull request by strega-nil**
Part of the localization effort

### CODE REVIEW

File: include/vcpkg/base/graphs.h [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
+#include <vcpkg/base/messages.h>
 
 #include <string>
 #include <unordered_map>
 #include <vector>
 
 namespace vcpkg::Graphs
 {
+    DECLARE_MESSAGE(GraphCycleDetected, (msg::value), "", "Cycle detected within graph at {value}:");
+    DECLARE_MESSAGE(GraphCycleDetectedElement, (msg::value), "{LOCKED}", "    {value}");
```
Comment (BillyONeal) [inline include/vcpkg/base/graphs.h:14]: Hmmm what is "LOCKED"?

I would argue this is locale-invariant?

**Review by BillyONeal — APPROVED** [review]

File: include/vcpkg/base/graphs.h (hunk 1 above)
Comment (strega-nil) [reply include/vcpkg/base/graphs.h:14]: yeah, `{LOCKED}` means "locale-invariant"

Comment (strega-nil-ms) [reply include/vcpkg/base/graphs.h:14]: I mean, it's fine but I'd rather not

Comment (ras0219-msft) [reply include/vcpkg/base/graphs.h:14]: In contrast I rather would avoid complicating our localization with trivial things like "add 4 spaces in front of this string". I think `unsafe_blah` is better and more declarative for this purpose.

**Pull request merged**